src/lib/constants.ts

```diff
@@ -427,3 +427,14 @@ export const chompyHats = [
 ] as const;
 
 export const gitHash = execSync('git rev-parse HEAD').toString().trim();
+
+export const toaPurpleItems = resolveItems([
```

Comment:
this array would fit more in the toa.ts to make it more streamlined with other uniqueArrays